### PR TITLE
Add test cases to check if docker, kubernetes, istio, openshift, knative artifacts are included in the distribution correctly.

### PR DESCRIPTION
**Pull request by hemikak**

### CODE REVIEW

File: ballerina-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                        </maven.version>
+                    </systemPropertyVariables>
+                    <suiteXmlFiles>
+                        <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
+                    </suiteXmlFiles>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+</project>
```
Comment (keizer619) [inline ballerina-test/pom.xml]: Shall we include the newline?

File: ballerina-test/pom.xml [hunk 2]
```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (keizer619) [inline ballerina-test/pom.xml]: Shall we include License header?

File: ballerina-test/src/test/resources/openshift/hello_world_oc.bal [hunk 3]
```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2019 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (keizer619) [inline ballerina-test/src/test/resources/openshift/hello_world_oc.bal]: This has to be 2020

File: ballerina-test/src/test/resources/kubernetes/hello_world_k8s.bal [hunk 4]
```diff
@@ -0,0 +1,23 @@
+import ballerina/http;
```
Comment (keizer619) [inline ballerina-test/src/test/resources/kubernetes/hello_world_k8s.bal]: Some of bal files has LICENSE, shall we add here as well?

File: ballerina-test/pom.xml (hunk 2 above)
Comment (hemikak) [reply ballerina-test/pom.xml]: Fixed with https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/6f025ca59e8db4293a7ca8655a149b66e3aef1d2

File: ballerina-test/src/test/resources/openshift/hello_world_oc.bal (hunk 3 above)
Comment (hemikak) [reply ballerina-test/src/test/resources/openshift/hello_world_oc.bal]: https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/96b01c79136f64c90671e0b7151493c75c047704

File: ballerina-test/src/test/resources/kubernetes/hello_world_k8s.bal (hunk 4 above)
Comment (hemikak) [reply ballerina-test/src/test/resources/kubernetes/hello_world_k8s.bal]: https://github.com/ballerina-platform/ballerina-distribution/pull/444/commits/96b01c79136f64c90671e0b7151493c75c047704

**Pull request merged**